config/routes.rb

```diff
@@ -11,4 +13,5 @@
   delete '/logout',  to: 'sessions#destroy'
   resources :users
   resources :account_activations, only: [:edit]
+  resources :password_resets,     only: [:new, :create, :edit, :update]
```

Comment:
- Use `except` instead of `only` when you have more than 3 elements in the array
- Use %i() instead of [] for large array of symbol